### DISCUSSION TITLE
Removing self hosted runner from deleted repository

### DISCUSSION THREAD

**Discussion by alexanderjuhn** [community/community#26297] [Actions]
<p>Hi everyone,</p>
<p>for testing purposes I setup a self hosted runner on a machine for a test repository.<br>
After some tests I deleted the repository without unregistering the runner first.</p>
<p>If I try to setup the runner for another repository I get the message that the runner is already registered and I should run config.sh remove first.<br>
If I run config.sh remove then I’m asked to input a removal token which I have not access to because the repository for that runner does not exist anymore.</p>
<p>Is there any way to unregister the runner without waiting for 30 days after which it is unregistered because of inactivity?</p>

**Comment by rukiaEnix** [comment, marked as answer]
<p>Hello,<br>
On the runner server just remove the whole actions-runner directory.<br>
Then do all the steps to add a new runner again, as shown on your github add runner instructions.</p>

**Comment by alexanderjuhn** [comment]
<p>Thank you, this solved my issue</p>
